blog/urls.py

```diff
@@ -0,0 +1,15 @@
+from blog.models import Commentary
+from blog.views import index, PostDetailView, CommentaryCreateView
+
+from django.urls import path
+
+app_name = "blog"
+
+urlpatterns = [
+    path("", index, name="index"),
+    path("posts/<int:id>", PostDetailView.as_view(), name="post-detail"),
```

Comment:
The URL pattern for 'post-detail' should ideally end with a trailing slash to follow Django's convention for URL design. Consider changing it to 'posts/<int:id>/' to ensure consistency and avoid potential issues with URL resolution.